google/cloud/forseti/services/inventory/storage.py

```diff
@@ -84,6 +85,7 @@ class InventoryIndex(BASE):
     inventory_index_warnings = Column(Text(16777215))
     inventory_index_errors = Column(Text())
     message = Column(Text())
+    notified_at_datetime = Column(DateTime())
```

Comment:
I asked this somewhere before, but what is this  `notified_at_datetime` used for?  I want to be sure that we are adding something here that we will use for sure.

Reply:
It captures the date/time at which the inventory summary was uploaded to GCS.